build.gradle

```diff
@@ -14,6 +14,7 @@ apply plugin: 'groovy'
 apply plugin: 'kotlin'
 apply plugin: 'maven'
 apply plugin: 'signing'
+apply plugin: 'java-gradle-plugin'
```

Comment:
I would be nice to use the plugins block, though that would be a separate PR.

Reply:
Agree. Let's keep this PR's scope small, though.